README.md

```diff
@@ -42,11 +42,17 @@ Currently, this server only supports hosting 1 room at a time.
 
 ## Usage
 
+*Note*: You must be running the Windows version of the game to use this server. If you are on Linux, use Proton.
+
 This is **NOT** meant to be used in any form of serious environment. This is an experimental testing server, and there is no way to safely configure a web browser to connect to this server currently.
 
 1. `npm install ws` to install the WebSockets NodeJS module.
-2. Generate a TLS certificate for the web server to use.
-3. Edit the top of `johnbox.js` to change accessibleHost to a host accessible by all players (e.g. public IP)
-    * This host must have a valid identity in the TLS certificate generated above
-4. `node johnbox` to start the server.
-5. Redirect the game to connect to your server. `jbg.config.jet` in each minigame folder has a `serverUrl` parameter.
\ No newline at end of file
+2. Generate a TLS certificate for the web server to use
+   * You can use [this tutorial](https://gist.github.com/cecilemuller/9492b848eb8fe46d462abeb26656c4f8) to generate these certificates (make sure to use the *Chrome, IE11 & Edge* method when trusting the new certificates, or your OS's specific way).
+   * If you are running this on a seperate server (not localhost), you should use a domain name and Let's Encrypt to access the server.
```

Comment:
I don't particularly like the idea of recommending people setting up domain names for this, especially not without a reverse proxy e.g. Nginx.

Reply:
My thinking as that it's the user's choice whether they want to host the server locally or on another server. And if they are hosting on another server, a domain name and a Let's Encrypt certificate is definitely the best way

Reply:
It's not.